ecs-init/config/common.go

```diff
@@ -44,17 +47,25 @@ const (
 	DefaultRegionName = "us-east-1"
 )
 
-// supportedRegions stores a list of regions that contains the agent bucket
-var supportedRegions = []string{"cn-north-1", "us-gov-west-1", DefaultRegionName}
+var partitionBucketMap = map[string]string{
+	"aws":        "us-east-1",
```

Comment:
You can use [these constants](https://github.com/aws/aws-sdk-go/blob/9a2fe34af9644afba4a1a1406966e78eb0e985af/aws/endpoints/defaults.go#L11-L13) and [these constants](https://github.com/aws/aws-sdk-go/blob/9a2fe34af9644afba4a1a1406966e78eb0e985af/aws/endpoints/defaults.go#L16-L44)

```go
var partitionBucketMap = map[string]string{
	endpoints.AwsPartitionID:      endpoints.UsEast1RegionID,
	endpoints.AwsCnPartitionID:    endpoints.CnNorth1RegionID,
	endpoints.AwsUsGovPartitionID: endpoints.UsGovWest1RegionID,
}
```